CMakeLists.txt

```diff
@@ -134,6 +134,9 @@ install_basic_package_files(${PROJECT_NAME}
   NO_CHECK_REQUIRED_COMPONENTS_MACRO
   DEPENDENCIES ${OSQP_EIGEN_EXPORTED_DEPENDENCIES})
 
+# Install package.xml to share
+install(FILES package.xml DESTINATION share/cmake/osqp-eigen)
```

Comment:
Just a curiosity, who needs this? I was just wondering as the CMake package name of this package is `OsqpEigen`, not `osqp-eigen` (even if the package is packed in conda-forge as `osqp-eigen`).

Reply:
We need it in some CI applications where OSQP -> OsqpEigen -> downstream packages are compiled in colcon/catkin workspaces